youtube/generic.md

```diff
@@ -1,50 +1,71 @@
 
-Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
+3. Open The Issue
 
-- NoSQL serverless database with DataStax, you will get $25 FREE credit every month http://links.eddiejaoude.io/t/bBM0STvCi2a4
-- Live streaming with Streamyard http://links.eddiejaoude.io/t/pYlzGjcta1a4
-- Cloud hosting with Digital Ocean, you will get $100 FREE credit http://links.eddiejaoude.io/t/EAaCRWhsnZlF
-- Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
-- Awesome backup platform, BackBlaze http://links.eddiejaoude.io/t/4rqTZjLbwmCp
+4. You get an invitation to the community by our GitHub Action 🎉
 
-## COMMUNITY SPONSORS
+---
+
+## Community Sponsors
 
-A BIG thank you to my GitHub Sponsors:
-***
 Stephen Mount http://github.com/stemount
+
 Nicholas Carrigan https://github.com/nhcarrigan
-***
 
-- Santosh Yadav https://github.com/santoshyadavdev
-- Andrew Cunliffe https://github.com/andrew-cunliffe
-- Alexander Stoichkov https://github.com/SashoStoichkov
-- Allan Regush https://github.com/AllanRegush
+Santosh Yadav https://github.com/santoshyadavdev
+
+Andrew Cunliffe https://github.com/andrew-cunliffe
+
+Alexander Stoichkov https://github.com/SashoStoichkov
+
+Allan Regush https://github.com/AllanRegush
+
+---
+
+## Service I Use and Recommend
+
+Some of these might be affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount too
+
+\- NoSQL serverless database with DataStax, you will get $25 FREE credit every month http://links.eddiejaoude.io/t/bBM0STvCi2a4
+\- Live streaming with Streamyard http://links.eddiejaoude.io/t/pYlzGjcta1a4
+\- Cloud hosting with Digital Ocean, you will get $100 FREE credit http://links.eddiejaoude.io/t/EAaCRWhsnZlF
+\- Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
+\- Awesome backup platform, BackBlaze http://links.eddiejaoude.io/t/4rqTZjLbwmCp
+
+---
+ 
+## Find me on socials
+
+Website: https://www.eddiejaoude.io 
+
+GitHub: https://github.com/eddiejaoude
+
+YouTube: https://youtube.com/eddiejaoude
+
+Twitter: https://twitter.com/eddiejaoude
 
-## FIND ME ON OTHER SOCIAL PLATFORMS
+Instagram: https://instagram.com/eddiejaoude
 
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
+LinkedIn: https://linkedin.com/in/eddiejaoude
 
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
+GitHub Community Org: https://github.com/EddieHubCommunity
 
-http://eddiejaoude.io 
-https://github.com/eddiejaoude
-http://youtube.com/eddiejaoude
-http://twitter.com/eddiejaoude
-http://instagram.com/eddiejaoude
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+## Further
```

Comment:
Maybe `miscellaneous` would be better instead?
```suggestion
## Miscellaneous
```

Reply:
Futher seems a little more friendly to me. Miscellaneous will be a little to much formal context to the actual text inside

Reply:
What about `more information`? Or `extras` or something like that? I feel `further` doesn't fit well there.

Reply:
> What about `more information`? Or `extras` or something like that? I feel `further` doesn't fit well there.

These words too aren't a fit here. But maybe **Meet our Community** will work?

Reply:
That sounds great! 😄

Reply:
Essentially, the Heading isn't really necessary
You could state it without a heading